content/blog/2022/05/2022-05-20-gsoc2022-projects-announcement.adoc

```diff
@@ -0,0 +1,61 @@
+sig:
+- gsoc
+description: >
+  Congratulations to the selected Jenkins 2022 GSoC project contributors.
+opengraph:
+  image: /images/gsoc/opengraph.png
+---
+
+image:/images/gsoc/jenkins-gsoc-logo_small.png[Jenkins GSoC, role=center, float=right]
+
+On behalf of the Jenkins GSoC team and mentors,
+We would like to welcome
+link:https://github.com/dheerajodha[Dheeraj Singh Jodha],
+link:https://github.com/Cr1t-GYM[Yiming Gong],
+link:https://github.com/hrushi20[Hrushikesh Rao], and
+link:https://github.com/vihaanthora[Vihaan Thora].
+They will be working on Google Summer of Code projects in the Jenkins organization,
+and they have already done some contributions.
+
+This year we have the following projects:
+
+* **Plugin Health Scoring System** -
+Provide a 'plugin health indicator' for plugin maintainers and Jenkins administrators.
+** Contributor: link:https://github.com/dheerajodha[Dheeraj Singh Jodha] from India
+** Mentors: link:/blog/authors/jleon[Jake Leon], link:https://github.com/alecharp[Adrien Lecharpentier], and link:https://github.com/ADI10HERO[Aditya Srivastava]
+
+* **Jenkinsfile Runner Action for GitHub Actions** -
+Implement a (near) feature-complete version of Jenkinsfile Runner Action for GitHub Actions.
+** Contributor: link:https://github.com/Cr1t-GYM[Yiming Gong] from Carnegie Mellon University, USA
+** Mentors: link:/blog/authors/krisstern[Kris Stern], link:https://github.com/AbhyudayaSharma[Abhyudaya Sharma], and link:https://github.com/imskr[Shubham Kumar]
+
+* **Automatic git cache maintenance on the controller** -
+Automate the maintenance of cached git repositories on the Jenkins controller.
+** Contributor: link:https://github.com/hrushi20[Hrushikesh Rao] from Jawaharlal Nehru Technological University, India
+** Mentors: link:https://github.com/markewaite[Mark Waite] and link:https://github.com/rishabhBudhouliya[Rishabh Budhouliya]
+
+* **Pipeline Step Documentation Generator Improvements** -
+"Enhance the Jenkins Pipeline documentation generator to produce better documentation for thousands of Pipeline developers".
+** Contributor: link:https://github.com/vihaanthora[Vihaan Thora] from Indian Institute of Technology, Indore, India
+** Mentors: link:https://github.com/kwhetstone[Kristin Whetstone], link:https://github.com/arpoch[Harshit Chopra], and
+link:https://github.com/koushartasneem[Tasneem Koushar]
+
+During the next weeks project teams will be reaching out to potential stakeholders in order to establish connections and
+to get comments regarding their project designs.
+If you are interested in the projects, please join discussions in the
+link:https://groups.google.com/forum/#!forum/jenkinsci-dev[Developer mailing lists] and project meetings once they get scheduled.
+Please also expect expect more detailed blogposts about the projects soon.
+
+If you are interested to know more about GSoC in Jenkins, you can find information, timeline and communication channels
+link:https://jenkins.io/projects/gsoc/[here].
```

Comment:
We prefer site local links without the URL so that contributors in development mode are not directed outside their development copy when they click links.
```suggestion
link:/projects/gsoc/[here].
```